lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_method.rb

```diff
@@ -168,11 +168,13 @@ def self.with_automation_env
     private_class_method :with_automation_env
 
     def self.process_ruby_method_results(return_code, msg, miq_request_id)
+      logger = ManageIQ::AutomationEngine::Logger.create_log_wrapper(:resource_id => miq_request_id)
```

Comment:
I wonder if we can simplify the logging even further. Moving a logger into the workspace is helpful, but also thinking about creating the log wrapper being an even simpler method.

```ruby
ManageIQ::AutomationEngine.logger(miq_request_id)
```

That is just an example, haven't thought through this properly.